lib/binary_to_decimal.rb

```diff
@@ -4,6 +4,56 @@
 # The least significant bit is at index 7.
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
-def binary_to_decimal(binary_array)
-  raise NotImplementedError
+# A method named `binary_to_decimal` that receives as input an array of size 8.
+# The array is randomly filled with 0’s and 1’s.
+# The most significant bit is at index 0.
+# The least significant bit is at index 7.
+# Calculate  and return the decimal value for this binary number using
+# the algorithm you devised in class.
+#
+array = []
+8.times do
+  array << rand(2)
+end
+
+def binary_to_decimal(array)
```

Comment:
This works, but doesn't scale well.  Could you think of a way to do this with a loop?